manifests/config.pp

```diff
@@ -6,7 +6,13 @@
 
   $version = $kibana4::package_ensure
 
-  file { "${kibana4::install_dir}/kibana-${version}/config/kibana.yml":
+  if $kibana4::use_official_repos {
```

Comment:
Can you make this use the if package_provide = 'package'. We use the official repo but a mirror as no external access and this PR hard sets the mirrors to use.